spec/factories/attachment_factory.rb

```diff
@@ -30,8 +30,13 @@
   factory :attachment do
     container factory: :work_package
     author factory: :user
-    sequence(:filename) { |n| "test#{n}.test" }
-    sequence(:disk_filename) { |n| "test#{n}.test" }
+    sequence(:file) do |n|
+      OpenProject::Files.create_uploaded_file name: "file-#{n}.test",
+                                              content_type: 'application/binary',
+                                              binary: true
+    end
+
+    content_type "application/binary"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.